src/components/dls/Popover/index.tsx

```diff
@@ -34,6 +34,7 @@ interface Props {
   triggerStyles?: string;
   contentStyles?: string;
   contentSideOffset?: number;
+  className?: string;
```

Comment:
Where are we using Popover in the PR?

Reply:
we no longer use it actually, removing this 👍